Documentation/gettingstarted/external-ips.rst

```diff
@@ -0,0 +1,83 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _external-ips:
+
+*****************************
+Kubernetes externalIPs (beta)
```

Comment:
Wouldn't it make sense to merge this doc with the NodePort GSG? Many parts seem to be identical between those two, so I'd create a section for the ExternalIP in the NodePort GSG.